frontend/webadmin/modules/frontend/src/main/resources/org/ovirt/engine/ui/frontend/AppErrors.properties

```diff
@@ -525,6 +525,8 @@ ACTION_TYPE_FAILED_STORAGE_CONNECTION_FOR_DOMAIN_NOT_EXIST=Cannot ${action} ${ty
 ACTION_TYPE_FAILED_STORAGE_CONNECTION_ID_EMPTY=Cannot ${action} ${type}. Storage connection id is empty.
 ACTION_TYPE_FAILED_STORAGE_CONNECTION_NOT_EXIST=Cannot ${action} ${type}. Storage connection doesn't exist.
 ACTION_TYPE_FAILED_UNSUPPORTED_ACTION_FOR_MANAGED_BLOCK_STORAGE_TYPE=Cannot ${action} ${type}.  Managed Block Storage does not support this operation.
+ACTION_TYPE_FAILED_UNSUPPORTED_ACTION_FROM_MANAGED_BLOCK_STORAGE_TYPE=Cannot ${action} ${type} from Managed Block Storage to Local SD. Managed Block Storage does not support this operation.
+ACTION_TYPE_FAILED_UNSUPPORTED_ACTION_TO_MANAGED_BLOCK_STORAGE_TYPE=Cannot ${action} ${type} from Local SD to Managed Block Storage. Managed Block Storage does support this operation.
```

Comment:
how about having a single error:
Cannot ${action} ${type}. Copying data between Local Storage and Managed Block Storage is not supported.